### DISCUSSION TITLE
WebView2 in WinUI2 sometimes raised CoreProcessFailed error

### DISCUSSION THREAD

**Discussion by supperwu** [MicrosoftEdge/WebView2Feedback#3906] [Q&A]
I'm trying to follow the [guide ](https://docs.microsoft.com/en-us/microsoft-edge/webview2/get-started/winui2)to use the WebView2 in WinUI 2, and sometimes I got the CoreProcessFailed error and sometimes the webview2 works fine.

Environment:
Win10 Enterprise 1909 with OS build:18363.1734
WebView2 runtime: 93.0.961.38
Microsoft.UI.Xaml: 2.7.0-prerelease.210827001

I register WebView2's events as below:
MyWebView.CoreWebView2Initialized += MyWebView_CoreWebView2Initialized;
MyWebView.CoreProcessFailed += MyWebView_CoreProcessFailed;
MyWebView.NavigationStarting += MyWebView_NavigationStarting;
MyWebView.NavigationCompleted += MyWebView_NavigationCompleted;

private void MyWebView_NavigationCompleted(WebView2 sender, CoreWebView2NavigationCompletedEventArgs args)
{
    Debug.WriteLine("Navigation completed.");
}

private void MyWebView_NavigationStarting(WebView2 sender, CoreWebView2NavigationStartingEventArgs args)
{
    Debug.WriteLine("Navigation starting.");
}

private void MyWebView_CoreWebView2Initialized(WebView2 sender, CoreWebView2InitializedEventArgs args)
{
    Debug.WriteLine("Initialized.");
}

private void MyWebView_CoreProcessFailed(WebView2 sender, CoreWebView2ProcessFailedEventArgs args)
{
    Debug.WriteLine("Process failed.");
    Debug.WriteLine($"Reason:{args.Reason}, exit code:{args.ExitCode}, failed kind:{args.ProcessFailedKind}");
}

And I try to navigate to a online page by clicking a button to change the WebView2's source.
Then I sometimes got the output like below, and the WebView2's background is black.

Initialized.
Navigation starting.
Process failed.
Reason:Unresponsive, exit code:259, failed kind:RenderProcessUnresponsive
Process failed.
Reason:Unresponsive, exit code:259, failed kind:RenderProcessUnresponsive
Process failed.
Reason:Unresponsive, exit code:259, failed kind:RenderProcessUnresponsive

What's I missing? What should I do? Any help would be appreciated, thank you in advance.

**Comment by Elisa-Adela** [comment]
Sometimes we also get a similar error:
'CoreWebView2_ProcessFailed: CoreWebView2_ProcessFailed event ExitCode: '259', Reason: 'Unexpected', ProcessDescription: '', ProcessFailedKind: 'BrowserProcessExited', FrameInfosForFailedProcess': 
WebView2 SDK 1.0.1185.39 and 100.0.1185.50 runtime
No crash dump available.

**Reply by DavidTynewyddBI** [reply]
Also seeing this, same exit code and also no dump available
